ldap/servers/slapd/daemon.c

```diff
@@ -2452,6 +2458,19 @@ createsignalpipe(void)
     return (0);
 }
 
+static int
+destroysignalpipe(void)
+{
+    int i;
```

Comment:
size_t i, and also inline with "for ( size_t i = 0; ...". This allows compiler optimisation over the iterator because the variable is constrained.